src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/main.py

```diff
@@ -256,14 +256,15 @@ def worker_thread(
 
     organizations = Organizations(
         role=boto3,
-        account_id=account_id
+        account_id=account_id,
+        cache=cache,
     )
     ou_id = organizations.get_parent_info().get("ou_parent_id")
 
     account_path = organizations.build_account_path(
-        ou_id,
-        [],  # Initial empty array to hold OU Path,
-        cache
+        ou_id=ou_id,
+        account_path=[],  # Initial empty array to hold OU Path
+        cache=cache,
```

Comment:
```suggestion
```

Can you also remove the cache parameter on line 495?

Reply:
Could you please remove the cache parameter in the other build_account_path call on line 491?